tests/test_command_line_ping3.py

```diff
@@ -7,8 +7,8 @@
 from unittest.mock import patch
 
 sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-import command_line_ping3  # noqa: linter (pycodestyle) should not lint this line.
-import errors  # noqa: linter (pycodestyle) should not lint this line.
+from ping3 import _main as command_line_ping3  # noqa: linter (pycodestyle) should not lint this line.
```

Comment:
Can we leave the filename of this one "command_line_ping3.py"? So we can just `from ping3 import command_line_ping3` and it looks more reasonable than `_main` for me.

Reply:
Fair enough. How about renaming it to "from ping3 import command_line" instead?

Reply:
Sounds great!